src/integrationTest/groovy/wooga/gradle/snyk/tasks/MonitorIntegrationSpec.groovy

```diff
@@ -27,6 +29,11 @@ import static com.wooga.gradle.test.PropertyUtils.toSetter
 
 class MonitorIntegrationSpec extends SnykCheckBaseIntegrationSpec<Monitor> {
 
+    @Override
+    String getOptionName() {
```

Comment:
Lets call this `getCommandName` or `getSubCommandName`

Reply:
Agreed.